outputs.tf

```diff
@@ -53,12 +53,7 @@ output "ecs_security_group" {
 
 output "ecs_cluster_id" {
   description = "ECS cluster id"
-  value       = module.ecs.ecs_cluster_id
-}
-
-output "ecs_cluster_arn" {
```

Comment:
lets leave this in for now so its not technically a "breaking change" - it will just be empty when the cluster is not created by the module. we can update the description to reflect with "ID of ECS cluster created"